Rmd/DESeq2_report.rnaseq.Rmd

```diff
@@ -718,7 +718,7 @@ This table shows the final list of samples that were used in the data analysis (
 if (any(grepl(x = colnames(DESeqDesign), pattern = "dose", ignore.case = T))) {
   doseCol <- grep(x = colnames(DESeqDesign), pattern = "dose", ignore.case = T)
   if(length(doseCol)>1){
-    doseCol <- doseCol[1]
+    doseCol <- doseCol[1] # This should be assigned explictly instead. Todo.
```

Comment:
Can you file an issue for this? Unless you're already working on it, I forget what you told me on Teams